test/integration/messaging.spec.ts

```diff
@@ -37,6 +37,30 @@ const condition = '"test0" in topics || ("test1" in topics && "test2" in topics)
 
 const invalidTopic = 'topic-$%#^';
 
+const message: admin.messaging.Message = {
+  data: {
+    foo: 'bar',
+  },
+  notification: {
+    title: 'Message title',
+    body: 'Message body',
+  },
+  android: {
+    restrictedPackageName: 'com.hkj.testing'
```

Comment:
hkj is not in the codebase yet, maybe put something generic here.

Reply:
Done